daemon/volumes.go

```diff
@@ -385,15 +385,14 @@ func copyExistingContents(source, destination string) error {
 // copyOwnership copies the permissions and uid:gid of the source file
 // into the destination file
 func copyOwnership(source, destination string) error {
-	var stat syscall.Stat_t
-
-	if err := syscall.Stat(source, &stat); err != nil {
+	stat, err := system.GetStat(source)
```

Comment:
@ahmetalpbalkan i am aware of that, i was just curious whether we could reuse system.Lstat instead of having a system.GetStat. If we don't want Lstat, then I would actually prefer renaming `system.GetStat()` to `system.Stat()`, but then we would need to do something with the `system.Stat` struct. Would it be possible to lowercase it?